src/main/java/core/basesyntax/generator/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.generator;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        for (Map.Entry<String, Integer> entry : Storage.fruits.entrySet()) {
+            report.append(entry.getKey()).append(",").append(entry.getValue()).append("\n");
```

Comment:
Ensure that the report format is consistent with any specified requirements. If there are none, consider adding a header to the report for clarity, such as 'Fruit,Quantity\n'.